src/main/java/ch/njol/skript/effects/EffEquip.java

```diff
@@ -82,7 +93,6 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	private static final boolean SUPPORTS_LLAMAS = Skript.classExists("org.bukkit.entity.Llama");
```

Comment:
```suggestion
```
Since the next update will drop 1.12 support, you don't need to check if llamas exist cause they were added in 1.11 (same goes with the `SUPPORTS_HORSES` and `NEW_HORSES`)